plugins/cros-ec/fu-cros-ec-firmware.h

```diff
@@ -7,7 +7,7 @@
 #pragma once
 
 #include "fu-firmware.h"
-#include "fu-fmap-firmware.h"
+#include "fu-fmap-firmware-private.h"
```

Comment:
mm this doesn't feel right that a plugin should rely upon a private header.  Remember plugin ABI is supposed to be public..

Reply:
I wasn't sure what to do here; exporting FuFmap and FuFmapArea C structs seemed like the wrong thing to do. Maybe what we had before was the least bad thing afterall?

Reply:
I've undone the -private change. If we do anything it shouldn't be part of this crash-fix bug.